tests/unit/test_property.py

```
@@ -0,0 +1,207 @@
+        try:
+            return importlib.import_module(name)
+        except ImportError as e:  # pragma: no cover - only runs on failure paths
+            last_exc = e
+            continue
+    # As a fallback, search sys.modules for any module already loaded that defines the symbols
+    for mod in list(sys.modules.values()):
+        if not isinstance(mod, types.ModuleType):
+            continue
+        if hasattr(mod, "GraphQLProperty") and hasattr(mod, "graphQlProperty"):
+            return mod
+    raise AssertionError(f"Could not import module under test via candidates: {candidates}. Last error: {last_exc!r}")
```

Ruff (0.13.2)

Avoid specifying long messages outside the exception class

(TRY003)